webrequest/requirements.txt

```diff
@@ -1,7 +1,9 @@
 Flask-JSONRPC==3.0.1
-Flask[async]==3.0.3
+Flask[async]==3.1.0
 python-dotenv==1.0.1
-pytest==8.3.3
+beautifulsoup4==4.12.3
+selenium==4.26.1
+webdriver-manager==4.0.2
```

Comment:
there are new packages here that I think shouldn't be

Reply:
Removed, maybe I trusted too much in the bot :D